web/themes/custom/liibarrio2020/templates/ds/liibarrio-card-secondary.html.twig

```diff
@@ -17,12 +17,12 @@
  */
 #}
 
-<{{ outer_wrapper }}{{ attributes.addClass('card', 'card-block', 'card-body', 'bg-secondary', 'text-white') }}>
+<{{ outer_wrapper }}{{ attributes.addClass('card', 'card-block', 'card-body', 'text-blue') }}>
```

Comment:
I don't think you need to add text-blue -- that should come from the links inside the card?